node_modules/oae-authentication/lib/api.js

```diff
@@ -87,7 +87,6 @@ var _configUpdate = function(tenantAlias) {
     }
 };
 
-ConfigAPI.on('cleared', _configUpdate);
```

Comment:
I figured if a globally configured key gets cleared it might have an impact on all strategies.